content/tr-tr/blog/2024-12-18-Aegisub_3.4.0_Released.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Aegisub 3.4.0 Yayınlandı
+contributors:
+  - CoffeeFlux

```

Comment:
imo. you can add yourself to the contributor list like was done for the [Chinese post](https://github.com/Aegisub/aegisite/blob/master/content/zh-cn/blog/2024-12-18-Aegisub_3.4.0_Released.md?plain=1#L5-L6)
Alternatively, we could also introduce a new frontmatter entry for `translators` and then adjust the template to show both separately.